### ISSUE TITLE
Provide a way to have hierarchical agendas

### ISSUE THREAD

**Issue by damithc** [PowerPointLabs/powerpointlabs#696]
Currently, all agenda items appear at the same level. Some users might have agenda levels at multiple levels.

**Comment by nevet** [comment]
Does it apply to all agendas? Or only bullet agenda? Could you give me an
example of hyerachical visual agenda?

On Friday, April 24, 2015, Damith C. Rajapakse <notifications@github.com>
wrote:

> Currently, all agenda items appear at the same level. Some users might
> have agenda levels at multiple levels.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PowerPointLabs/powerpointlabs/issues/696>.
>


-- 
Best Regards,
Huang Da (Mr.)
National University of Singapore, Computer Engineering, Year 4
+65 86552730

**Comment by damithc** [comment]
I guess it applies to bullet point agenda only.

**Comment by nevet** [comment]
I think this issue is duplicate with #655.

**Comment by nevet** [comment]
Please see the discussion in #655.

**Comment by Skaty** [comment]
Can we do it based on a special character (suggestion: star or dash) that is prepended in the section name? That way, it is not as hacky as referring to the first slide.

Another method is to have a context menu button when right clicking on the bullet points, that will allow users to sync the levels up.

**Comment by damithc** [comment]
> Can we do it based on a special character (suggestion: star or dash) that is prepended in the section name? That way, it is not as hacky as referring to the first slide.

Yes, a dash followed by space can be taken as level 2 topic. two dashes followed by space can be taken as level 3 topic.
